Ps 116 adding .env file

JafariM opened the pull request.

JafariM committed:
add a condtion so loggedIn user can not see the login and signup buttons

JafariM committed:
add .env file with variable,update logout component to use it

JafariM committed:
add env varialbe to singIn component

JafariM committed:
add env variable to userRegister component

JafariM committed:
add env variable to App.js file

JafariM committed:
add env variable to index.js file

JafariM committed:
add env variable to axiosClient file

JafariM committed:
add a comment on mirageSever file

JafariM committed:
add documentation on how use the env variable

akosasante commented on src/App.js:
```diff
@@ -7,23 +7,10 @@ import { Navigate, Route, Routes } from 'react-router-dom';
-const URL = 'http://localhost:8000/api/v1/';
-
 function App() {
   const [message, setMessage] = useState('');
```
We can remove this as well, it's no longer used here

akosasante commented on src/App.js:
```diff
@@ -7,23 +7,10 @@ import { Navigate, Route, Routes } from 'react-router-dom';
 import Navbar from './components/Navbar';
 import { AuthStatus, useAuth } from '@akosasante/react-auth-context';
 
```
We can also remove `useEffect` and `getAllData` from the imports as they're not used anymore

JafariM replied:
I removed them

akosasante reviewed (approved):
LGTM! :+1: 

I changed the base branch to the `connect login` one because this PR seemed to include commits from there. Since we don't want to accidentally merge those until that branch is fixed, I changed the base branch to reflect that for now

JafariM committed:
did some cleanup in App.js

JafariM commented:
> LGTM! 👍
> 
> I changed the base branch to the `connect login` one because this PR seemed to include commits from there. Since we don't want to accidentally merge those until that branch is fixed, I changed the base branch to reflect that for now

Good, thank you
